examples/ethercat/ecat_load_firmware.py

```diff
@@ -3,10 +3,10 @@
 from ingenialink.ethercat.network import EthercatNetwork
 
 
-def ecat_load_fw():
+def ecat_load_fw() -> None:
     net = EthercatNetwork("\\Device\\NPF_{192D1D2F-C684-467D-A637-EC07BD434A63}")
 
-    net.load_firmware(fw_file='../../resources/firmware/cap-net-e_0.7.1.lfu')
+    net.load_firmware(fw_file="../../resources/firmware/cap-net-e_0.7.1.lfu", boot_in_app=False)
```

Comment:
This could be dangerous. I think we should add a comment explaining how to determine the correct value for this parameter.

Reply:
I've instead changed the actual function to have "False" as a default if the parameter is not provided since the use of the parameter is described in the docstring of the function.